src/Application.vala

```diff
@@ -48,8 +51,63 @@ public class Mail.Application : Gtk.Application {
         set_accels_for_action ("app.quit", {"<Control>q"});
     }
 
-    public override void open (File[] files, string hint) {
-        activate ();
+    public override int command_line (ApplicationCommandLine command_line) {
+        activate();
+
+        string[] argv = command_line.get_arguments ();
+
+        // The only arguments we support are mailto: URLs passed in by the OS. See RFC 2368 for
+        // details. We handle the most commonly used fields.
+        foreach (var mailtoUri in argv[1:argv.length]) {
```

Comment:
elementary code style is for variables to use `snake_case` instead of `camelCase`

Reply:
Ha, was trying to do that, but old habits die hard.